### PR TITLE
Increase python to version 3.10

### PR DESCRIPTION
**Pull request by firasm**
~As mentioned by @Bluesy1 [here](https://github.com/open-resources/problem_bank_scripts/pull/60#discussion_r1259066146), the path needs to be a string to split it properly.~

### CODE REVIEW

**Comment by firasm** [comment]
Resolves #64

**Comment by firasm** [comment]
superseded by #66

File: src/problem_bank_scripts/problem_bank_scripts.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 ## Topic Validation
 topic_list = set()
-path = pathlib.Path().resolve().as_posix()
+path = pathlib.Path().resolve()
```
Comment (Bluesy1) [inline src/problem_bank_scripts/problem_bank_scripts.py:36]: Still need to stringify the path here :)

File: src/problem_bank_scripts/problem_bank_scripts.py [hunk 2]
```diff
@@ -34,6 +34,7 @@
 ## Topic Validation
 topic_list = set()
 path = pathlib.Path().resolve()
```
Comment (firasm) [inline src/problem_bank_scripts/problem_bank_scripts.py:36]:
```suggestion
path = pathlib.Path().resolve().as_posix()
```

**Pull request merged**